tests/src/util/playgrounds/unique.dev.ts

```diff
@@ -84,7 +85,32 @@ class ArrangeGroup {
       }
     }
 
-    await Promise.all(transactions);
+    await Promise.all(transactions).catch(e => {});
+    
+    //#region TODO remove this region, when nonce problem will be solved
+    const checkBalances = async () => {
+      let isSuccess = true;
+      for (let i = 0; i < balances.length; i++) {
+        const balance = await this.helper.balance.getSubstrate(accounts[i].address);
+        if (balance !== balances[i] * tokenNominal) {
```

Comment:
Not sure for that case (Where all accounts are new), but more correct way is balance `>=` instead of `!==`